src/platformdirs/unix.py

```diff
@@ -0,0 +1,129 @@
+import os
+import sys
+
+from .api import PlatformDirsABC
+
+
+class Unix(PlatformDirsABC):
+    """
+    On Unix/Linux, we follow the
+    `XDG Basedir Spec <https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html>`_. The spec allows
+    overriding directories with environment variables. The examples show are the default values, alongside the name of
+    the environment variable that overrides them. Makes use of the
+    `appname <platformdirs.api.PlatformDirsABC.appname>`,
+    `version <platformdirs.api.PlatformDirsABC.version>`,
+    `multipath <platformdirs.api.PlatformDirsABC.multipath>`,
+    `opinion <platformdirs.api.PlatformDirsABC.opinion>`.
+    """
+
+    @classmethod
+    def is_active(cls) -> bool:
+        """:return: a check to detect if Unix/Linux platform is currently active"""
+        return sys.platform not in ("win32", "darwin")
+
+    @property
+    def user_data_dir(self) -> str:
+        """
+        :return: data directory tied to the user, e.g. ``~/.local/share/$appname/$version`` or
+         ``$XDG_DATA_HOME/$appname/$version``
+        """
+        if "XDG_DATA_HOME" in os.environ:
+            path = os.environ["XDG_DATA_HOME"]
+        else:
+            path = os.path.expanduser("~/.local/share")
+        return self._path_with_app_name_version(path)
+
+    def _path_with_app_name_version(self, of: str) -> str:
+        params = []
+        if self.appname:
+            params.append(self.appname)
+            if self.version:
+                params.append(self.version)
+        return os.path.join(of, *params)
+
+    @property
+    def site_data_dir(self) -> str:
+        """
+        :return: data directories shared by users (if `multipath <platformdirs.api.PlatformDirsABC.multipath>` is
+         enabled and ``XDG_DATA_DIR`` is set and a multi path the response is also a multi path separated by the OS
+         path separator), e.g. ``/usr/local/share/$appname/$version`` or ``/usr/share/$appname/$version``
+        """
+        # XDG default for $XDG_DATA_DIRS; only first, if multipath is False
+        if "XDG_DATA_DIRS" in os.environ:
+            path = os.environ["XDG_DATA_DIRS"]
+        else:
+            path = f"/usr/local/share{os.pathsep}/usr/share"
+        return self._with_multi_path(path)
```

Comment:
why not work with split paths here and only join the result

Reply:
Can you detail? I've just kept the existing method, but we can switch if you propose something better.